recipes/markermag/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "MarkerMAG" %}
+{% set version = "1.1.26" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/MarkerMAG-{{ version }}.tar.gz
+  sha256: a5a87c5dd319130fb47f7105759e1e48c3906f3e0ce8533241cb2a1d75d97882
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
have a look at https://docs.conda.io/projects/conda-build/en/latest/user-guide/wheel-files.html and https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html. Usually, we use  " script: python -m pip install --no-deps --ignore-installed ."

"We strongly recommend using the --no-deps option in the pip install command.

If you run pip install without the --no-deps option, pip will often install dependencies in your conda recipe and those dependencies will become part of your package. This wastes space in the package and increases the risk of file overlap, file clobbering, and broken packages.